plugins/xsts/hu.bme.mit.gamma.lowlevel.xsts.transformation/src/hu/bme/mit/gamma/lowlevel/xsts/transformation/XstsUtils.xtend

```diff
@@ -0,0 +1,77 @@
+package hu.bme.mit.gamma.lowlevel.xsts.transformation
```

Comment:
This class could be moved to `hu.bme.mit.gamma.xsts.model/src/hu/bme/mit/gamma/xsts/util`